src/main/java/Event.java

```diff
@@ -0,0 +1,29 @@
+public class Event extends Task  {

```

Comment:
Noticed that java docs are missing for all the classes and methods, might be good to add it in!

Reply:
Great! Will add.